doctrine/doctrine-bundle/1.12/config/packages/prod/doctrine.yaml

```diff
@@ -0,0 +1,3 @@
+doctrine:
+    orm:
+        auto_generate_proxy_classes: false
```

Comment:
What about using `%kernel.debug%` as the value in `doctrine/doctrine-bundle/1.12/config/packages/doctrine.yaml` instead?